### PR TITLE
feat(snippet): Add 'Hide liked songs card' snippet

### PR DESCRIPTION
**Pull request by MetteDE**
Just a simple CSS snippet to hide the liked Songs card in the "Your Library" task, because I often find it useless. Maybe some other people will find this helpful. 

previews

### CODE REVIEW

**Comment by CharlieS1103** [comment]
Seems all good to me, is the image alright @theRealPadster

**Comment by Tetrax-10** [comment]
its better if you use `collection-collectionEntityHeroCard-likedSongs` instead of `main-heroCard-card`. who knows spotify may add a diff hero card (big card) for events may be.

**Comment by MetteDE** [comment]
Yea I tried multiple different classes but there was no difference in the outcome so I chose main-heroCard-card because of simplicity

**Comment by theRealPadster** [comment]
> Seems all good to me, is the image alright @theRealPadster

I'd recommend adding a red x through it, so it's clear it removes that section.

**Comment by MetteDE** [comment]
Like this? 

hide-likedSongs-card

File: packages/marketplace/src/resources/snippets.ts [hunk 1]
```diff
@@ -164,7 +164,7 @@ export default [
   {
     "title": "Hide liked songs card",
     "description": "Hides the sometimes unfitting liked Songs card in the Your Libary tab",
-    "code":".collection-collectionEntityHeroCard-likedSongs{ display: none; }",
-    "preview":"resources/assets/snippets/hide-likedSongs-card.png"
+    "code": ".collection-collectionEntityHeroCard-likedSongs{ display: none; }",
+    "preview": "resources/assets/snippets/hide-likedSongs-card.png"
```
Comment (theRealPadster) [inline packages/marketplace/src/resources/snippets.ts:168]: Missing trailing comma

**Comment by theRealPadster** [comment]
Yep that's better 👍

**Review by theRealPadster — CHANGES_REQUESTED** [review]

**Comment by MetteDE** [comment]
Sorry, I have a lot do to in the next few days and kinda new to this github stuff. I hope, its complete now!

File: packages/marketplace/src/resources/snippets.ts [hunk 2]
```diff
@@ -161,4 +161,10 @@ export default [
     "code": `@keyframes rotating { from { transform: rotate(0deg); } to { transform: rotate(360deg); }} .cover-art { animation: rotating 2s linear infinite; clip-path: circle(50% at 50% 50%);}`,
     "preview": "resources/assets/snippets/rotating-coverart",
   },
+  {
+    "title": "Hide liked songs card",
+    "description": "Hides the sometimes unfitting liked Songs card in the Your Libary tab",
+    "code": ".collection-collectionEntityHeroCard-likedSongs{ display: none; }",
+    "preview": "resources/assets/snippets/hide-likedSongs-card.png",
+  }
```
Comment (theRealPadster) [inline packages/marketplace/src/resources/snippets.ts:169]: I think this is the last change. You should just be able to commit from this suggestion :) 
```suggestion
  },
```

**Review by theRealPadster — CHANGES_REQUESTED** [review]

**Comment by MetteDE** [comment]
Thanks a lot! :D

**Comment by Tetrax-10** [comment]
> 

ofcource there wont be any diff. `main-heroCard-card` is a general class name for big cards in your lib. currently there is only 1 hero card which is the liked songs. spotify may include a hero card in future who know.

**Pull request merged**